### PR TITLE
Moved footer out of page content containe

### PR DESCRIPTION
**Pull request by bdemers**
Moved footer out of page content container

Lots of lint and small fixes:
- menu was reusing IDs
- iframe attributes
- moved "menu" template into header.ftl
- added comments on tags opened in header and closed in footer

### CODE REVIEW

File: jbake/templates/index.ftl [hunk 1]
```diff
@@ -110,70 +110,64 @@
...
+                <a class="btn btn-outline-primary" href="<#if (content.rootpath)??>${content.rootpath}</#if>/${(currentPageNumber==2)?then('', currentPageNumber-1)}">Previous</a>
               </#if>
               <#if (currentPageNumber > 1 && currentPageNumber != numberOfPages)>
-              <li class="page-item"><a class="page-link" href="${currentPageNumber}">${currentPageNumber}</a></li>
+                <a class="btn btn-outline-primary " href="${currentPageNumber}">${currentPageNumber}</a>
               </#if>
               <#if (currentPageNumber < numberOfPages)>
-              <li class="page-item"><a class="page-link" href="<#if (content.rootpath)??>${content.rootpath}</#if>${currentPageNumber + 1}">Next</a></li>
+                <a class="btn btn-outline-primary" href="<#if (content.rootpath)??>${content.rootpath}</#if>${currentPageNumber + 1}">Next</a>
               </#if>
```
Comment (bmarwell) [inline jbake/templates/index.ftl:137]: Oh yeah that also works. Anything wrong with following https://getbootstrap.com/docs/5.0/components/pagination/ though? Did it not work for you? (style-wise)

Comment (bmarwell) [reply jbake/templates/index.ftl:137]: I need to fix those links btw :(

File: jbake/templates/index.ftl [hunk 2]
```diff
@@ -110,70 +110,64 @@
...
+          <h3 class="card-title">Latest news:</h3>
         </div>
 
-        <div class="panel-body">
+        <div class="card-body">
         <#list posts as post>
         <#if (post.status == "published"  && post?index >= (currentPageNumber-1) * config.index_posts_per_page?eval && post?index < currentPageNumber * config.index_posts_per_page?eval)>
           <div>
-            <a href="${post.uri}"><small><h4>${post.title}</h4></small></a>
+            <a href="${post.uri}"><small>${post.title}</small></a>
```
Comment (bmarwell) [inline jbake/templates/index.ftl:122]: Important links to other pages should have a heading from what I know, at least all major SEO sites are still doing this, too (and wordpress if you enable snippets and jekyll and jbake in  their default templates...).

**Review by bmarwell — APPROVED** [review]
Wow, gerat PR! Thanks!

Just two nits (added as comments) and one request: In case you moved the pictures from the old site, can you use `git mv` so they get deleted in the old path? I think this would help us keep track of what we have done so far.

Thanks! 👍🏻

File: jbake/templates/index.ftl (hunk 1 above)
Comment (bdemers) [reply jbake/templates/index.ftl:137]: Oh nice! I didn't see that component! I'll tweak it!

File: jbake/templates/index.ftl (hunk 2 above)
Comment (bdemers) [reply jbake/templates/index.ftl:122]: ahh, ok, we might just need to style them a bit to reduce the font size

**Comment by bdemers** [comment]
> Wow, gerat PR! Thanks!
> 
> Just two nits (added as comments) and one request: In case you moved the pictures from the old site, can you use `git mv` so they get deleted in the old path? I think this would help us keep track of what we have done so far.
> 
> Thanks! 👍🏻

Oh yeah, I forgot about that, will do!

File: jbake/templates/index.ftl (hunk 1 above)
Comment (bdemers) [reply jbake/templates/index.ftl:137]: Actually, I think I was just trying to fix the centering and didn't realize it was an existing component 🤦

**Comment by bdemers** [comment]
@bmarwell the `git mv` of the images required a rebase/force-push, but I think I address all three issues, take a look.

**Pull request merged**